conference/2024.md

```diff
@@ -24,7 +24,23 @@ and dedicated time for discussions and coding.
 
 Links to previous FEniCS conferences can be found [here](index.md).
 
-More information about the conference will appear here soon.
+## Abstract Submission
+
+Please submit your abstract by **April 5th** by filling in the [abstract submission form](pages_2024/registration_form.md).
+
```

Comment:
Add about the conference proceedings here? Feel free to use info from our shared doc.